python/examples/agents/granite.py

```diff
@@ -7,32 +7,34 @@
 from beeai_framework.memory.unconstrained_memory import UnconstrainedMemory
 from beeai_framework.tools.search import DuckDuckGoSearchTool
 from beeai_framework.tools.weather.openmeteo import OpenMeteoTool
-from examples.helpers.io import prompt_input
+from examples.helpers.io import ConsoleReader
 
 
 async def main() -> None:
-    chat_model: ChatModel = await ChatModel.from_name("ollama:granite3.1-dense:8b")
+    chat_model: ChatModel = ChatModel.from_name("ollama:granite3.1-dense:8b")
```

Comment:
`: ChatModel` is not need I would say

Reply:
I was say we should leave it in examples because it clarifies what type the var is to the reader. But you are correct that is not necessary